timesketch/lib/analyzers/sessionizer_test.py

```diff
@@ -0,0 +1,236 @@
+        analyser = sessionizer.SessionizerSketchPlugin(index, sketch_id)
+        self.assertIsInstance(analyser, SessionizerSketchPlugin)
+        self.assertEqual(index, analyser.index_name)
+        self.assertEqual(sketch_id, analyser.sketch.id)
+
+    @mock.patch(u'timesketch.lib.analyzers.interface.ElasticsearchDataStore',
+                MockDataStore)
+    def test_same_session(self):
+        """Test multiple events in the same session are allocated correctly."""
+        with mock.patch.object(SessionizerSketchPlugin,
+                               'event_stream',
+                               return_value=_create_mock_event(
+                                   0, 2, time_diffs=[200])):
+            index = 'test_index'
+            sketch_id = 1
+            analyser = sessionizer.SessionizerSketchPlugin(index, sketch_id)
+            message = analyser.run()
+            self.assertEqual(
+                message,
+                'Sessionizing completed, number of session created: 1')
+
+            ds = MockDataStore("test", 0)
+            event1 = (ds.get_event('test_index', '0', stored_events=True))
+            self.assertEqual(event1['_source']['session_number'], 1)
+            event2 = (ds.get_event('test_index', '1', stored_events=True))
+            self.assertEqual(event2['_source']['session_number'], 1)
+
+    @mock.patch(u'timesketch.lib.analyzers.interface.ElasticsearchDataStore',
+                MockDataStore)
+    def test_diff_session(self):
+        """Test multiple events in different sessions are allocated
+        correctly."""
+        with mock.patch.object(SessionizerSketchPlugin,
+                               'event_stream',
+                               return_value=_create_mock_event(
+                                   0, 2, time_diffs=[400000000])):
+            index = 'test_index'
+            sketch_id = 1
+            analyser = sessionizer.SessionizerSketchPlugin(index, sketch_id)
+            message = analyser.run()
+            self.assertEqual(
+                message,
+                'Sessionizing completed, number of session created: 2')
+
+            ds = MockDataStore("test", 0)
+            event1 = (ds.get_event('test_index', '0', stored_events=True))
+            self.assertEqual(event1['_source']['session_number'], 1)
+            event2 = (ds.get_event('test_index', '1', stored_events=True))
+            self.assertEqual(event2['_source']['session_number'], 2)
+
+    @mock.patch(u'timesketch.lib.analyzers.interface.ElasticsearchDataStore',
+                MockDataStore)
+    def test_multiple_sessions(self):
+        """Test multiple events, two of which are in the same session and
+        one in a different session."""
+        with mock.patch.object(SessionizerSketchPlugin,
+                               'event_stream',
+                               return_value=_create_mock_event(
+                                   0, 3, time_diffs=[300000000, 400000000])):
```

Comment:
I would use a value a bit smaller than `300000000` here so as to make the test more clear (you're already testing the edge case in the test after this)